app.json

```diff
@@ -0,0 +1,7 @@
+{
+  "name": "kepler",
+  "buildpacks": [{
+    "url": "https://github.com/dulaccc/heroku-buildpack-geodjango.git#1.1"
```

Comment:
Yeah, I'm not worried about setting it manually as I know that works, I was just trying to do it through config. Based on the log messages I see during the build I'm not convinced it's using this buildpack. Maybe this is only used for github deploys?